gini/gini-vision-lib-ios
# Migrate to Swift 3 [WIP]

## Information

This is a WIP merge to track the difference between develop and Swift 3.

## How to test

Checkout project and run example app. There should be no differences in behaviour and UI compared to other Swift versions.

## Merge information

Should *not* be merged until future handling of Swift versions is determined. 

## YouTrack Tickets

MOBILE-657
